### ISSUE TITLE
Default z-index

### ISSUE THREAD

**Issue by nomego** [cwdoh/paper-fab-menu#3]
We're using other things with overlays in our application but paper-fab-menu has a default `z-index: 10000` which causes it to always be on top.

I'd rather see that you put the paper-fab-menu in an element with the correct z-index already, and the menu just takes care of itself when activated.

For a quick fix, I'd recommend moving the rule

``` css
.container {
  z-index: 10000;
}
```

down to a 

``` css
.container.show {
  z-index: 10000;
}
```

rule to make sure it only elevates above everything once activated.

I guess this is somewhat related to #2 as well.

**Comment by cwdoh** [comment]
Applied to 16558348ecc4d3091328f5b4e55ca505f2476f4a. It seemed to be considerable before merging that. I'm not convinced that other issues can be appeared or not. Thanks for the advice. :)

**Comment by nomego** [comment]
Yes that branch/commit fixes this issue, but not #2 :)

I wouldn't mind seeing this branch merged to master.

**Comment by cwdoh** [comment]
Merged. :)
